libraries net stress tests fail git checkout on Docker NanoServer

https://dev.azure.com/dnceng/public/_build/results?buildId=552319&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=1d3f18c3-4e0e-535e-19e9-0e4b922e42dd
https://dev.azure.com/dnceng/public/_build/results?buildId=552318&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=1d3f18c3-4e0e-535e-19e9-0e4b922e42dd

```
git checkout --progress --force 5aff3becf9d9e9f3bc77ee6e3442b3ec0e3d1605
fatal: reference is not a tree: 5aff3becf9d9e9f3bc77ee6e3442b3ec0e3d1605
##[warning]Git checkout failed on shallow repository, this might because of git fetch with depth '1' doesn't include the checkout commit '5aff3becf9d9e9f3bc77ee6e3442b3ec0e3d1605'. Please reference documentation (http://go.microsoft.com/fwlink/?LinkId=829603)
##[error]Git checkout failed with exit code: 128
Finishing: Checkout dotnet/runtime@master to s
```

--- a/eng/pipelines/libraries/stress/http.yml
+++ b/eng/pipelines/libraries/stress/http.yml
@@ -56,7 +56,7 @@ jobs:
   steps:
   - checkout: self
     clean: true
-    fetchDepth: 1
+    fetchDepth: 5
     lfs: false
   
   - pwsh: |

--- a/eng/pipelines/libraries/stress/ssl.yml
+++ b/eng/pipelines/libraries/stress/ssl.yml
@@ -57,7 +57,7 @@ jobs:
   steps:
   - checkout: self
     clean: true
-    fetchDepth: 1
+    fetchDepth: 5
     lfs: false
 
   - pwsh: |

